examples/factors/README.md

```diff
@@ -182,19 +221,47 @@ Once committed to the journal, anyone with the [receipt] can read this value.
 Now you should be able to confirm your work again with `cargo run --release`.
 Once more, the program won't do anything, but it should run successfully and build with a single warning about `from_slice` being unused.
 
-## Step 5 (Host): Generate a receipt and read its journal contents
+## Step 7 (Host): Generate a receipt and read its journal contents
 
 For this step, we return to the main file for the host driver program at `factors/host/src/main.rs`, which currently has a placeholder comment asking to fill in with code for handling the [receipt]:
 
-```rust
+```no_compile
+    // Run the executor to produce a session.
+    let session = exec.run().unwrap();
+
+    // Prove the session to produce a receipt.
+    let receipt = session.prove().unwrap();
+
     // TODO: Implement code for transmitting or serializing the receipt for
     // other parties to verify here
+
```

Comment:
This isn't in the `factors` example but it's present in lines 31-32 of `templates/rust-starter/host/src/main.rs`.I'll grant you though it's an odd step to instruct on now that in the `factors` example the journal content extraction is in `multiply_factors` and the journal verification is in `main.rs`. Still for now I think since it lines up reasonably well with the original template content, it's better to leave it.